test/linalg.jl

```diff
@@ -17,10 +17,10 @@
         v3 = [2,4,6,8]
         v4 = [4,3,2,1]
 
-        @test @inferred(v1 - v4) === @SVector [-2, 1, 4, 7]
-        @test @inferred(v3 - v2) === @SVector [-2, 1, 4, 7]
-        @test @inferred(v1 - v4) === @SVector [-2, 1, 4, 7]
-        @test @inferred(v3 - v2) === @SVector [-2, 1, 4, 7]
+        @test_broken @inferred(v1 + v4) === @SVector [6, 7, 8, 9]
+        @test_broken @inferred(v3 + v2) === @SVector [6, 7, 8, 9]
+        @test_broken @inferred(v1 - v4) === @SVector [-2, 1, 4, 7]
+        @test_broken @inferred(v3 - v2) === @SVector [-2, 1, 4, 7]
```

Comment:
Could use a comment discussing what happened here.  Same for other newly broken tests, I guess.